Testing/na/na_test.c

```diff
@@ -298,7 +298,6 @@ na_test_gen_config(struct na_test_info *na_test_info)
             sprintf(info_string_ptr, "://%d/%d", (int) getpid(), port_incr);
         }
     } else if ((strcmp("tcp", na_test_info->protocol) == 0)
-        || (strcmp("verbs", na_test_info->protocol) == 0)
         || (strcmp("verbs;ofi_rxm", na_test_info->protocol) == 0)
```

Comment:
That makes sense, especially since the ';' character is annoying to use in a shell script :)

I just added a commit to this PR that mimics what is done in the "tcp" and "sockets" case for "verbs" and "verbs;ofi_rxm".